serverless/admin_guide/serverless-kafka-admin.adoc

```diff
@@ -0,0 +1,27 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-kafka-admin"]
+= Knative Kafka
+include::modules/common-attributes.adoc[]
+:context: serverless-kafka-admin
+
+toc::[]
+
+In addition to the Knative Eventing components that are provided as part of a core {ServerlessProductName} installation, the `KnativeKafka` custom resource (CR) can be installed by cluster administrators.
```

Comment:
Optional rephrasing to avoid passive voice
```suggestion
In addition to the Knative Eventing components that are provided as part of a core {ServerlessProductName} installation, cluster administrators can install the `KnativeKafka` custom resource (CR).
```

Reply:
For sure, thanks!